apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx

```diff
@@ -301,11 +301,40 @@ export const CommonPluggableComponentForm: FunctionComponent<CommonPluggableComp
 
         }
 
-    }
+    };
+
+    const changeUserIdInClaimCheckboxLabelBasedOnValue = (key: string, values: Map<string, FormValue>) => {
```

Comment:
As a practice, add doc comments on a method so that any other developer can identify the functionality of the function.